configs/webpack.docs.dev.config.js

```diff
@@ -219,7 +219,12 @@ var docsConfig = {
             'process.env': {
                 'ENV': '"development"'
             }
+<<<<<<< HEAD
```

Comment:
This is a merge file - please resolve. If you have any trouble with merges, let me know and I can describe the workflow.